data-reconciliation-app/test/e2e-test/src/host.ts

```diff
@@ -0,0 +1,59 @@
+import { exec, execSync } from 'child_process';
+
+
+const buildCommand = 'npm run build';
+const startHostCommand = '/opt/ccf_virtual/bin/sandbox.sh --js-app-bundle ./dist/ --initial-member-count 3 --initial-user-count 0 --constitution-dir ./governance/constitution'
+
+export default class Host {
+
+    public static async start(): Promise<void> {
+        //
+        console.log(`📀 [Running] - Build App`);
+        await this.run(buildCommand);
+
+        console.log(`\n📀 [Running] - Start Host\n`);
+        await this.run(startHostCommand);
+
+        console.log(`✅ [Done] - Testing Environment Ready!`);
+    }
+
+    public static stop() {
+        console.log(`\n📀 [Running] - Stop Host\n`);
+
+        /**
+         * Sandbox script spins up the host in a child process using python
+         * We query the python processes and kill it
+         * This makes sure the host is stopped and the sandbox is cleaned up
+         * This also makes the e2e test script idempotent
+         */
+        execSync('kill -9 $(pgrep python)');
```

Comment:
Is there a nicer way to do this than killing python? Will you not have the shell script still running in a broken state?

I ask as I did this originally in the bash script and found side-effects of killing python

Reply:
As we discussed in the standup, this can be looked at it in another PR. I have created #142  for that.